arviz/data/io_numpyro.py

```diff
@@ -91,7 +92,12 @@ def arbitrary_element(dct):
                     for i, v in enumerate(tree_flatten_samples)
                 }
             self._samples = samples
-            self.nchains, self.ndraws = posterior.num_chains, posterior.num_samples
+            self.nchains, self.ndraws, self.nthinning = (
+                posterior.num_chains,
+                posterior.num_samples,
+                posterior.thinning,
+            )
+            self.ndraws //= self.nthinning
```

Comment:
```suggestion
            self.nchains, self.ndraws = (
                posterior.num_chains,
                posterior.num_samples // posterior.thinning,
            )
```

as we don't use `self.nthinning` anywhere else, I would define `self.ndraws` directly doing the division